roles/ovirt-set-mac-pools/README.md

```diff
@@ -0,0 +1,48 @@
+oVirt Mac pools
+=================
+
+The `ovirt-set-mac-pools` role is used to set up oVirt mac pools.
+
+Requirements
+------------
+
+ * oVirt Python SDK version 4
+ * Ansible version 2.3
+
+Role Variables
+--------------
+
+| Name                     | Default value         | Description                          |
+|--------------------------|-----------------------|--------------------------------------|
+| allow_duplicates         | UNDEF                 | If (true) allow a MAC address to be used multiple times in a pool.
```

Comment:
I think default is false in your role, not by ovirt engine, so UNDEF is not true also